src/components/layout/productCard/style.scss

```diff
@@ -61,3 +61,11 @@
 .slick-slide {
   display: inline-block;
 }
+
+.categoriesContainer .slick-slide > div {
+  width: 100%;
+}
+
+.slick-slide > div {
```

Comment:
Can you give those `divs` a className? Using direct descentant selector is kinda open to cause issues in case we wanna add another elements to that container. 

This was Louis' explanation to better understand it: [here](https://github.com/ReCoded-Org/istanbul-capstone-ecommerce/pull/27#discussion_r462207154)

Reply:
I tried it on my own after our meeting and it seems like that `div` is a default wrapper coming with `Slider` component, which we cannot add a className as far as I understood. Please just drop a comment above it so that we can understand it when we look at it later.